src/uct/api/uct.h

```diff
@@ -519,6 +519,11 @@ enum uct_flush_flags {
                                                  error state, and it becomes
                                                  unusable for send operations
                                                  and should be destroyed. */
+    UCT_FLUSH_FLAG_REMOTE   = UCS_BIT(1)    /**< Guarantees that the target
+                                                 buffer is updated. Effective
+                                                 for remote write operations
+                                                 such as RDMA write or
```

Comment:
Effective for remote memory write and remote atomic operations

Reply:
updated

Reply:
per UCX conf call: "flush to point of visibility"